core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -22,8 +25,10 @@ public class JdbcConfig {
       PREFIX + "prepared_statements_pool.max_open";
 
   public static final String ISOLATION_LEVEL = PREFIX + "isolation_level";
+  /** @deprecated As of 5.0, will be removed. Use {@link #METADATA_SCHEMA} instead. */
+  @Deprecated public static final String TABLE_METADATA_SCHEMA = PREFIX + "table_metadata.schema";
 
-  public static final String TABLE_METADATA_SCHEMA = PREFIX + "table_metadata.schema";
+  public static final String METADATA_SCHEMA = PREFIX + "metadata.schema";
```

Comment:
The config `table_metadata.schema` is renamed to `metadata.schema` as this schema now contains the metadata table and namespace table.